google/cloud/storage/bucket_metadata.cc

```diff
@@ -51,15 +51,20 @@ std::ostream& operator<<(std::ostream& os, BucketPolicyOnly const& rhs) {
 
 std::ostream& operator<<(std::ostream& os, BucketIamConfiguration const& rhs) {
   os << "BucketIamConfiguration={";
+  char const* sep = "";
+  if (rhs.public_access_prevention.has_value()) {
+    os << "public_access_prevention=" << *rhs.public_access_prevention;
```

Comment:
Opt: Perhaps we should stream `sep` here, to make the position of this element non-special.

Reply:
Done